### PR TITLE
Reword dmd.auto-ref-adjacent.dd

### PR DESCRIPTION
**Pull request by Bolpat**
Also, what about `auto ref` function returns? Those weren't mentioned.

### CODE REVIEW

**Review by dkorpel — APPROVED** [review]

**Comment by dkorpel** [comment]
> Also, what about `auto ref` function returns? Those weren't mentioned.

Still needs to be implemented

File: changelog/dmd.auto-ref-adjacent.dd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-`auto ref` variables must have adjacent keywords `auto` and `ref`
+Keywords `auto` and `ref` must be adjacent
 
 It's now deprecated to declare `auto ref` parameters without putting those two keywords next to eachother.
```
Comment (ntrel) [inline changelog/dmd.auto-ref-adjacent.dd:3]:
```suggestion
It's now deprecated to declare `auto ref` parameters without putting those two keywords next to each other.
```

**Comment by ntrel** [comment]
The ldc buildkite error is unrelated (possibly can be fixed by rebasing).

**Review by RazvanN7 — APPROVED** [review]

**Review by WalterBright — APPROVED** [review]

**Review by ntrel — APPROVED** [review]

**Comment by thewilsonator** [comment]
what's with the circle failure?

**Pull request merged**